cloudformation/slack-router-cfn-template.js

```diff
@@ -0,0 +1,99 @@
+const cf = require('@mapbox/cloudfriend')
+
+const Resources = {
+  SlackRouterApi: {
+    Type: 'AWS::ApiGatewayV2::Api',
+    Properties: {
+      Name: 'slack-router-api',
+      ProtocolType: 'HTTP',
+    },
+  },
+  SlackToLambdaIntegration: {
+    Type: 'AWS::ApiGatewayV2::Integration',
+    Properties: {
+      ApiId: cf.ref('SlackRouterApi'),
+      Description: 'Integration for Slack Slash Commands',
+      IntegrationType: 'AWS_PROXY',
+      IntegrationUri: cf.join('', [
+        'arn:',
+        cf.partition,
+        ':apigateway:',
+        cf.region,
+        ':lambda:path/2015-03-31/functions/',
+        cf.getAtt('SlackRouterLambda', 'Arn'),
+        '/invocations',
+      ]),
+      PayloadFormatVersion: '2.0',
+    },
+  },
+  SlackToLambdaRoute: {
+    Type: 'AWS::ApiGatewayV2::Route',
+    Properties: {
+      ApiId: cf.ref('SlackRouterApi'),
+      AuthorizationType: 'NONE',
```

Comment:
Can you explain why you chose `NONE` over the other options available here? Are the other options not available to implement at the moment?

Reply:
The other option is using [JWT Tokens](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-jwt-authorizer.html). Since we're going to be validating all requests that comes in using the Slack secret I wasn't sure if we needed this. I specified `NONE ` though to explicitly state this in case we want to change this.

@ramyaragupathy what do you think? I'm not sure what HOT policy is or what is best practice.

Reply:
We're good to go with the current implementation with slack secrets - wouldn't want to introduce another layer of complexity